configure-pull-request/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Configure Pull Request
+
+description: Sets the parameters of an existing Pull Request at once
+
+inputs:
+  github-token:
+    description: A GitHub personal access token with write access to the project
```

Comment:
I'd suggest adding explicit types for the inputs (e.g., `type: string`): https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-of-onworkflow_callinputs

Currently, GH ignores that in runtime, but this behavior may be useful in the future and provides a self-documenting code.

Reply:
Input type seems not to be allowed for composite action. I will try.

Reply:
ah I didn't know. Then let's not do it.

Reply:
I have added the types for the inputs as suggested (it works, you were rights)